delta/service/src/main/scala/ch/epfl/bluebrain/nexus/delta/service/resolvers/ResolversDeletion.scala

```diff
@@ -0,0 +1,41 @@
+package ch.epfl.bluebrain.nexus.delta.service.resolvers
+
+import ch.epfl.bluebrain.nexus.delta.sdk.Resolvers
+import ch.epfl.bluebrain.nexus.delta.sdk.ResourcesDeletion.{CurrentEvents, ProjectScopedResourcesDeletion, StopActor}
+import ch.epfl.bluebrain.nexus.delta.sdk.model.ResourcesDeletionProgress.{CachesDeleted, ResourcesDataDeleted}
+import ch.epfl.bluebrain.nexus.delta.sdk.model.projects.ProjectRef
+import ch.epfl.bluebrain.nexus.delta.sdk.model.resolvers.ResolverEvent
+import ch.epfl.bluebrain.nexus.delta.service.resolvers.ResolversImpl.{ResolversAggregate, ResolversCache}
+import ch.epfl.bluebrain.nexus.delta.sourcing.DatabaseCleanup
+import monix.bio.Task
+
+final class ResolversDeletion private (
+    cache: ResolversCache,
+    stopActor: StopActor,
+    currentEvents: CurrentEvents[ResolverEvent],
+    dbCleanup: DatabaseCleanup
+) extends ProjectScopedResourcesDeletion(stopActor, currentEvents, dbCleanup, Resolvers.moduleType)(_.id) {
+
+  override def deleteData(projectRef: ProjectRef): Task[ResourcesDataDeleted] =
```

Comment:
I think we should rename this method as it is not actually deleting anything. Something like `freeResources` would make more sense, as this is what it's actually doing for most resource types(elasticsearch and blazegraph views)